presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an ``IndexOutOfBoundException`` during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Fix a regression in cpu time introduced in 0.253 for queries using :func:`element_at` for maps.
+* Add fragment result caching support for ``UNNEST`` queries.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``. Enabling this property can help with investigating GC issues on the coordinator by allowing us to debug whether stats collection uses a lot of memory.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When ``query_limit_spill_enabled`` is set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.
+
+Hive Changes
+____________
+* Fix a bug where the files would not be sorted when inserting into bucketed sorted tables with Glue.
+* Add support for validating the values returned from the partition cache with the actual value from Metastore. This can be enabled by setting the configuration property ``hive.partition-cache-validation-percentage``. The default value is 0.0 which means there is no validation.
```

Comment:
Re: the second line - the default value is 0.0 - your previous sentence says "This can be enabled by " - that indicates that this is disabled by default. Had it been enabled by default - the sentence would have read - "This can be disabled by" - and hence it is redundant and can be removed.